ccp/index_reader.py

```diff
@@ -3,13 +3,23 @@
 creates the Jenkins pipeline projects from entries of index.
 """
 
+import exceptions
+import re
 import subprocess
 import sys
 import yaml
 
 from glob import glob
 
 
+class InvalidPipelineName(exceptions.Exception):
```

Comment:
Should we create a separate file for exceptions? We might add more exceptions in future now that we've become serious about testing. In that case we can do something like `from ccp_exceptions import InvalidPipelineName`. 

Another reason for this is that we will only keep increasing the size of py files if we add all into one. 

This is just a comment, I leave the final call on you @navidshaikh